python/desc/twinkles/sprinkler.py

```diff
@@ -175,14 +175,13 @@ def sprinkle(self):
                     row['galaxyAgn_magNorm'] = np.nan
                     row['galaxyAgn_sedFilename'] = None
                     #Now insert desired Bulge properties
-                    row['galaxyBulge_sedFilename'] = self.LRG_name
+                    row['galaxyBulge_sedFilename'] = newlens['lens_sed']
                     row['galaxyBulge_redshift'] = newlens['ZLENS']
                     row['galaxyDisk_redshift'] = newlens['ZLENS']
                     row['galaxyAgn_redshift'] = newlens['ZLENS']
                     row['galaxyBulge_magNorm'] = matchBase().calcMagNorm([newlens['APMAG_I']], self.LRG, self.bandpassDict) #Changed from i band to imsimband
-                    newlens['REFF'] = 1.0 #Hard coded for now. See issue in OM10 github.
-                    row['galaxyBulge_majorAxis'] = radiansFromArcsec(newlens['REFF'])
-                    row['galaxyBulge_minorAxis'] = radiansFromArcsec(newlens['REFF'] * (1 - newlens['ELLIP']))
+                    row['galaxyBulge_majorAxis'] = radiansFromArcsec(newlens['REFF'] / np.sqrt(1 - newlens['ELLIP']))
```

Comment:
Concerning the Reff. I decided to use the values from CATSIM for the galaxies we matched to and not use these values. When looking at the notebook predictions for Reff in the test set the errors were very large compared to the errors we found in stellar mass (see cell 11 and 12 in the notebook).